Start-FalkorDB.sh

```
@@ -0,0 +1 @@
+docker run -d -p 6379:6379 -it --rm falkordb/falkordb
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)